src/main/java/com/opentok/Archive.java

```diff
@@ -252,6 +257,14 @@ public String getPassword() {
         return password;
     }
 
+    /**
+     * The MultiArchiveTag is the tag of the archive, with which to start, this is unique for each archive
+     * within session. If this was set in the request, this response will include a new archive ID.
```

Comment:
This is incorrect. Please see the description in the REST API docs: https://tokbox.com/developer/rest/.